mxcubeweb/routes/beamline.py

```diff
@@ -182,13 +182,13 @@ def beamline_abort_action(name):
 
         :param str name: Owner / Actuator of the process/action to abort
 
-        Replies with status code 200 on success and 520 on exceptions.
```

Comment:
520 does not seem to be an official HTTP error code (it's used by Cloudflare, apparently). I think 500 is more appropriate?

Reply:
Agreed